user/src/routes/update-user.ts

```diff
@@ -37,7 +36,7 @@ router.patch(
     }
 
     if (password && password !== "") {
-      const existingUser = await User.findOne({ email });
```

Comment:
change the argument from `email` to `name` in the case that the user changes their email in the security panel on the dashboard so the `email` value can be changed but the `name` value still hold to the same value for checking purpose